js/apps/account-ui/src/account-security/SSHKeys.tsx

```diff
@@ -0,0 +1,287 @@
+  const [sshKeys, setSshKeys] = useState<string[]>();
+  const [openModal, setOpenModal] = useState(false);
+  const [key, setKey] = useState(0);
+  const refresh = () => setKey(key + 1);
+  const [deleteSSHKeyIndex, setDeleteSSHKeyIndex] = useState(-1);
+  usePromise((signal) => getSSHKeys({ signal, context }), setSshKeys, [key]);
+
+  if (!sshKeys) {
+    return <Spinner />;
+  }
+
+  const saveSSHKeys = async (sshKeys: string[], type: string) => {
+    try {
+      await updateSHHKeys(context, sshKeys);
+      refresh();
+      addAlert(t(type === "add" ? "sshKeyAddSuccess" : "sshKeyDeleteSuccess"));
+    } catch (error) {
+      addError(t(type === "add" ? "sshKeyAddError" : "sshKeyAddSuccess"));
+    }
+  };
+
+  return (
+    <Page title={t("sshKeys")} description={t("sshKeysDescrption")}>
+      <ConfirmationModal
+        deleteSSHKeyIndex={deleteSSHKeyIndex}
+        close={() => {
+          setDeleteSSHKeyIndex(-1);
+        }}
+        deleteSSHKey={(index: number) => {
+          const array = [...sshKeys];
+          array.splice(index, 1);
+          saveSSHKeys(array, "delete");
+          setDeleteSSHKeyIndex(-1);
+        }}
+      />
+      <SSHKeyModal
+        openModal={openModal}
+        sshKeys={sshKeys}
+        close={() => {
+          setOpenModal(false);
+        }}
+        save={(keyValue: string) => {
+          sshKeys.push(keyValue);
+          saveSSHKeys([...sshKeys], "add");
+          setOpenModal(false);
+        }}
+      />
+      <Split hasGutter className="pf-v5-u-mb-lg">
+        <SplitItem isFilled></SplitItem>
+        <SplitItem>
+          <Button
+            variant="primary"
+            icon={<PlusIcon />}
+            iconPosition="right"
+            onClick={() => setOpenModal(true)}
+          >
+            {t("addNew")}
+          </Button>
+        </SplitItem>
+      </Split>
+      <DataList className="ssh-keys-list" aria-label={t("sshKeys")}>
+        <DataListItem>
+          {sshKeys.map((key, index) => (
+            <DataListItemRow key={key}>
+              <DataListContent
+                aria-label="ssh-keys-content"
+                className="pf-v5-u-flex-grow-1"
+              >
+                <Grid hasGutter>
+                  <GridItem span={1}>
+                    <KeyIcon />
+                    <Badge isRead>SSH</Badge>
+                  </GridItem>
+                  <GridItem sm={8} md={9} span={10}>
+                    <span className="pf-u-mr-md">{key}</span>
+                  </GridItem>
+                  <GridItem
+                    className="pf-v5-u-text-align-right"
+                    sm={3}
+                    md={2}
+                    span={1}
+                  >
+                    <Button
+                      variant="danger"
+                      onClick={() => {
+                        setDeleteSSHKeyIndex(index);
+                      }}
+                    >
+                      Delete
```

Comment:
This needs to be translated
```ts
  const { t } = useTranslation();
```
```suggestion
                      {t("delete")}
```